torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -865,12 +892,27 @@ def local_state_dict(self, *args: Any, **kwargs: Any) -> Any:
         with self.state_dict_type(StateDictType.LOCAL_STATE_DICT):
             return self.state_dict(*args, **kwargs)
 
+    def full_state_dict(self, *args: Any, **kwargs: Any) -> Any:
```

Comment:
synced up offline, we can remove full_state_dict and load_full_state_dict interfaces